pkg/netpol/eval/ingressanalyzer/ingress_analyzer_test.go

```diff
@@ -0,0 +1,98 @@
+package ingressanalyzer
+
+import (
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	v1 "k8s.io/api/core/v1"
+
+	"github.com/np-guard/netpol-analyzer/pkg/netpol/eval"
+	"github.com/np-guard/netpol-analyzer/pkg/netpol/internal/testutils"
+	"github.com/np-guard/netpol-analyzer/pkg/netpol/logger"
+	"github.com/np-guard/netpol-analyzer/pkg/netpol/scan"
+)
+
+// global scanner object for testing
+var scanner = scan.NewResourcesScanner(logger.NewDefaultLogger(), false, filepath.WalkDir)
+
+func TestIngressAnalyzerWithRoutes(t *testing.T) {
+	routesNamespace := "frontend"
+	path := filepath.Join(testutils.GetTestsDirFromInternalSubDir(), "acs_security_frontend_demos")
```

Comment:
`GetTestsDirFromInternalSubDir()` this was added in `../internal/testutils/utils.go` since the regular `GetTestsDir()` supports only 3 level-ups while more is needed here.
if we want to keep the directory of `ingressanalyzer` we may update the `GetTestsDir()` to support a given level-ups